README.md

```diff
@@ -47,6 +47,9 @@ Then, run `python3 main.py`.
 At first run, you will be asked to setup Trakt and Plex access.
 Follow the instructions, your credentials and API keys will be stored in
 `.env` and `.pytrakt.json` files.
+
+If you have 2 Factor Authentication enabled on Plex, you can append the code to your password.
```

Comment:
include example perhaps?

Reply:
I'm not really sure what examples I can give. You literally just add the 2fa code to the end of your password when logging in.

Reply:
is there documentation of where you got this from? add such link?

Reply:
https://github.com/pkkid/python-plexapi/issues/609

https://support.plex.tv/articles/two-factor-authentication/
See "Old Third-Party Apps & Tools"

I guess we could add `<password><verification code>`?

Reply:
yes, do so, and here's the direct link:
- https://support.plex.tv/articles/two-factor-authentication/#toc-1:~:text=Old%20Third%2DParty%20Apps%20%26%20Tools

Reply:
Created followup:
- https://github.com/Taxel/PlexTraktSync/pull/125